packages/core/src/presets/torbox.ts

```
@@ -0,0 +1,159 @@
+        constants.DEBRID_STREAM_TYPE,
+        constants.USENET_STREAM_TYPE,
+      ],
+      SUPPORTED_RESOURCES: supportedResources,
+    };
+  }
+
+  static async generateAddons(
+    userData: UserData,
+    options: Record<string, any>
+  ): Promise<Addon[]> {
+    return [this.generateAddon(userData, options)];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)